profile_library/aqara/CL-L02D/model.json

```
@@ -0,0 +1,18 @@
+{
+  "created_at": "2025-01-18T11:56:26.347610",
+  "measure_description": "Measured with 6.85W (average over 300s) dummy load. Brightness 1, light still off. Standby power is 0.36W for this device in total but due to limitations divided between the two lights.",
+  "measure_device": "Shelly PM Mini Gen3 (S3PM-001PCEU16)",
+  "measure_method": "script",
+  "measure_settings": {
+    "SAMPLE_COUNT": 2,
+    "SLEEP_TIME": 3,
+    "VERSION": "v1.17.1:docker"
+  },
+  "name": "Ceiling light T1M (CL-L02D)",
```

```suggestion
  "name": "Ceiling light T1M",
```

Model id don't have to be repeated in the name